web/src/app/workplace/services/task.service.ts

```diff
@@ -1,18 +1,26 @@
-import { Task } from '../models/task';
-import { Observable } from 'rxjs/Observable';
-import { HttpClient, HttpHeaders } from '@angular/common/http';
-import { Injectable } from '@angular/core';
-import { environment } from 'app/../environments/environment';
+import {Task} from '../models/task';
+import {Observable} from 'rxjs/Observable';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {environment} from 'app/../environments/environment';
 
 @Injectable()
 export class TaskService {
   url = environment.taskanaRestUrl + '/v1/tasks';
 
+  httpOptions = {
```

Comment:
Remove authorization headers from services, I have created an interceptor for share module, and workplace module which overrides the default one.
[This applies to all authorization headers hardcoded]